content/md/what-is/understand-data-quality.md

```diff
@@ -29,14 +29,15 @@ From the `data quality insights` panel, by clicking on the "Display all missing
 # How is consistency calculated?
 
 The **consistency grade** is calculated on several criteria. The mix of the following criteria helps the PIM determine the grade for consistency:
-- Spell check: localizable text and text areas are checked for spelling mistakes.
+- Spell check: localizable text and text areas, attribute labels and attribute options are checked for spelling mistakes.
```

Comment:
consistency: you sometimes write spell check and sometimes spell-check. I recommend the first

Reply:
thanks!